photon-api/src/main/scala/com/linkedin/photon/ml/estimators/GameEstimator.scala

```diff
@@ -244,39 +254,104 @@ class GameEstimator(val sc: SparkContext, implicit val logger: Logger) extends P
       !ignoreThreshold || initialModelOpt.isDefined,
       "'Ignore threshold for new models' flag set but no initial model provided for warm-start")
 
-    // Partial retraining and warm-start training require an initial GAME model to be provided as input
-    val coordinatesToTrain = (initialModelOpt, retrainModelCoordsOpt) match {
-      case (Some(initModel), Some(retrainModelCoords)) =>
+    // Warm-start, partial re-training, and incremental training are mutually exclusive.
```

Comment:
Previously warm-start and partial-retraining are coupled. Now the dependency is decoupled, right?

Reply:
No. They are mutually exclusive. Updated the comments to make it clearer.